pkg/api/config.go

```diff
@@ -114,6 +114,10 @@ func validateBuildRootImageConfiguration(fieldRoot string, input *BuildRootImage
 		return nil
 	}
 
+	if input.ImageStreamTagReference != nil && len(input.ImageStreamTagReference.Cluster) == 0 {
+		return []error{fmt.Errorf("%s.image_stream_tag: no cluster defined", fieldRoot)}
```

Comment:
Its completely pointless that `validateBuildRootImageConfiguration` returns a slice of error rather than a single error.

Can we define a `var errs []error` and append all errors we find rather than returning them?

Reply:
Done.